### PR TITLE
newrelic-fluent-bit-output: fix aarch64 build

### PR DESCRIPTION
**Pull request by imjasonh**
We want to use the default gcc toolchain instead of overriding it, as the upstream build does.

Tested myself with:

```
make ARCH=aarch64 packages/newrelic-fluent-bit-output BUILDWORLD=no
```

cc @developer-guy FYI -- I wouldn't have expected to hit this before merging, just something to keep an eye out for in the future.

### CODE REVIEW

File: newrelic-fluent-bit-output/fix-aarch64.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
++++ b/Makefile
+@@ -6,7 +6,7 @@ linux/amd64:
+ 	CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -buildmode=c-shared -o out_newrelic-linux-amd64-${VERSION}.so .
+ 
+ linux/arm64:
+-	CGO_ENABLED=1 GOOS=linux GOARCH=arm64 CC=aarch64-linux-gnu-gcc CXX=aarch64-linux-gnu-g++ go build -buildmode=c-shared -o out_newrelic-linux-arm64-${VERSION}.so .
++	CGO_ENABLED=1 GOOS=linux GOARCH=arm64 go build -buildmode=c-shared -o out_newrelic-linux-arm64-${VERSION}.so .
+ 
+ linux/arm/v7:
+ 	CGO_ENABLED=1 GOOS=linux GOARCH=arm CC=arm-linux-gnueabihf-gcc CXX=arm-linux-gnueabihf-g++ go build -buildmode=c-shared -o out_newrelic-linux-arm-${VERSION}.so .
```
Comment (imjasonh) [inline newrelic-fluent-bit-output/fix-aarch64.patch:13]: Do we want to do our future selves a favor and remove this for arm/v7 too?

Comment (kaniini) [reply newrelic-fluent-bit-output/fix-aarch64.patch:13]: Yes, remove it for all of them tbh

**Review by luhring — APPROVED** [review]

**Comment by developer-guy** [comment]
oh, thanks @imjasonh, this is really a good example! We'll be more careful next time.

**Comment by imjasonh** [comment]
> oh, thanks @imjasonh, this is really a good example! We'll be more careful next time. 

I think you were perfectly careful! This is a weird case, I've never seen this before.

**Pull request merged**